setup.py

```diff
@@ -3,7 +3,7 @@
 
 __author__ = 'Martin Uhrin'
 __license__ = 'GPLv3 and MIT'
-__contributors__ = 'Sebastiaan P. Huber'
+__contributors__ = ['Sebastiaan P. Huber', 'Jusong Yu', 'Sonia Collaud']
```

Comment:
@unkcpz prefers to be referred to as "Jason Yu". Please correct me if I'm wrong

Reply:
@unkcpz How would you like your name?  I used the version from the AiiDA paper but if you prefer Jason I will use that.

Reply:
I'd like to be referred as "Jason Yu" here @muhrin . Thanks for pointing this out @sphuber .

Reply:
You got it buddy. Changed.